tests/foreman/virtwho/api/test_kubevirt.py

```diff
@@ -53,18 +53,19 @@ def virtwho_config(form_data, target_sat):
 
 
 @pytest.fixture(autouse=True)
-def clean_host(form_data, target_sat):
+def delete_host(form_data, target_sat):
```

Comment:
> So first yield and then do this procedure to run it in post part!
@jyejare That's really a good suggestion, because the delete_host fixture have in several classes, later,  will add the delete host in the fixture teardown. and remove the delete_host fixture. I have add this optimization in my tasks list.